docs/semgrep-app/findings.md

```diff
@@ -0,0 +1,161 @@
+
+
+![Screenshot of Semgrep App Findings page ](../img/app-findings-overview.png)<br />
+
+A finding is the core result of Semgrep's analysis. Findings are generated when a Semgrep rule matches a piece of code. 
+
+Types of findings:
+
+* anti-patterns
+* security vulnerabilities, business, or logic bugs
+* audits for dangerous function use
+* reviewing authentication or authorization logic
+* custom pattern matches based on your own custom rules
+
+
+After a finding is generated, developers may:
+
+* View the Semgrep rule and the matching code.
+* Fix the issue detected by the finding.
+* Ignore the finding.
+* Comment on the finding.
+* Create a Jira ticket from the finding (for Enterprise/Team Tier users.)
+
+
+The Semgrep App Finding page displays findings across all projects connected to Semgrep App. It is updated after every scan. Scans are initiated through your CI/CD pipeline, such as GitHub actions. The retention period of these findings varies based on your organization’s tier:
+
+| Retention period | Tier availability |
+| ---------------  | ----------------- |
+| 30-day findings retention | Community (Free) |
+| 1-year findings retention | Team/Enterprise |
+
+## Viewing data in the Findings page
+
+### Navigating to the Findings page
+
+1. Sign in to Semgrep App.
+2. The *Findings* page can be found on the left sidebar.
+
+### Understanding the Findings data
+
+The Findings page displays the following fields: 
+
+| Field       | Description | 
+| ----------  | ------------ | 
+| **Severity**    | The impact of a finding. Possible values: Low, Medium, High,and Critical. |
+| **Finding**     | The file, line, and branch of the pattern match in the code. Clicking on this field brings you to the exact location in the codebase. |
+| **Project**     | The name of the project codebase where the finding was found. Clicking on this field brings you to the project’s URL. |
+| **Rule**        | The name of the rule matched with the code. Clicking on this field brings you to the rule’s entry in the [rule registry](https://semgrep.dev/r). |
+| **Introduced**  | The time when this finding was first found. Clicking on the header row sorts the table by latest or earliest findings. |
+| **Status**      | Refers to a finding’s triage state. |
+
+## Triaging findings
+
+Triaging means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
+
+Semgrep App assists in the triage process through the use of *comments* and *triage states*:
+
+| Triage state | Description |
+| -----------  | ------------ |
+| **Open** | Open findings require action, such as rewriting the code for vulnerabilities, or refactoring the code. Findings are open by default. |
+| **Ignored** | Findings that are ignored will not be acted upon. This may be a false positive or deprioritized issue. |
```

Comment:
Maybe we can add that findings can be Ignored via the App (and then they appear as Ignored - App) or in the code via the nosemgrep comment / .semgrepignore file (and then they appear as Ignored - Code)

Reply:
Nice catch! To keep the findings page in scope, and to present the user with more information, I'm linking to the "ignore-findings" article in the docs.